test/deploy-tests-manifest.json

```diff
@@ -3,7 +3,8 @@
   "suites": {
     "test/e2e/app-dir/actions/app-action.test.ts": {
       "failed": [
-        "app-dir action handling fetch actions should invalidate client cache when path is revalidated"
+        "app-dir action handling fetch actions should invalidate client cache when path is revalidated",
+        "app-dir action handling fetch actions should handle revalidateTag"
```

Comment:
This test has been flaking, so I'm going to be deflaking it in the next PR in this stack

Reply:
Seems to be timing related. Ended up pushing this in the same PR, commit: [3984916](https://github.com/vercel/next.js/pull/70726/commits/39849166557d3e495f1539df6f1d720246357887)